client/src/test/java/org/spine3/time/ZoneOffsetsShould.java

```diff
@@ -46,8 +47,10 @@ public void has_private_constructor() {
 
     @Test
     public void create_default_instance_according_to_place() {
-        final int currentOffset = TimeZone.getDefault().getRawOffset()/(int)MILLIS_PER_SECOND;
+        final int currentOffset = timeZone.getRawOffset()/(int)MILLIS_PER_SECOND;
```

Comment:
We repeat this calculation over and over again. Why don't  create a utility method?

E.g. `getTimezoneZeconds()`.